src/app/pages/home/home.page.html

```diff
@@ -10,41 +10,64 @@
 </ion-header>
 
 <ion-content>
-    <ion-card class="welcome-card">
-      <ion-img style="margin:0 auto; width: 80%" src="assets/aerogear.png"></ion-img>
-      <ion-card-header>
-        <ion-card-title>AeroGear Mobile Services Showcase</ion-card-title>
-      </ion-card-header>
-      <ion-card-content>
-        <p>Sample application that integrates all AeroGear Mobile Services
-        </p>
-        <ion-card-subtitle>Enabled services:</ion-card-subtitle>
-        <ion-badge color="warning" >No services provisioned</ion-badge>
-      </ion-card-content>
-    </ion-card>
-    <ion-list lines="none">
-      <ion-list-header>
-        <ion-label >Menu</ion-label>
-      </ion-list-header>
-      <ion-item routerDirection='root' routerLink="docs">
-        <ion-icon slot="start" color="medium" name="book"></ion-icon>
-        <ion-label>View Documentation</ion-label>
-      </ion-item>
-      <ion-item routerDirection='root' routerLink="tasks">
-        <ion-icon slot="start" color="medium" name="build"></ion-icon>
-        <ion-label>Manage Tasks</ion-label>
-      </ion-item>
-      <ion-item routerDirection='root' routerLink="profile">
-        <ion-icon slot="start" color="medium" name="person"></ion-icon>
-        <ion-label>User profile</ion-label>
-      </ion-item>
-      <ion-item routerDirection='root' routerLink="files">
-        <ion-icon slot="start" color="medium" name="document"></ion-icon>
-        <ion-label>Documents</ion-label>
-      </ion-item>
-      <ion-item routerDirection='root' routerLink="security">
-        <ion-icon slot="start" color="medium" name="settings"></ion-icon>
-        <ion-label>Device Security</ion-label>
-      </ion-item>
-    </ion-list>
-</ion-content>
+  <ion-card class="welcome-card">
+    <ion-img class="logo" src="assets/aerogear.png"></ion-img>
+    <ion-card-header>
+      <ion-card-title text-center>AeroGear Mobile Services Showcase Application</ion-card-title>
+    </ion-card-header>
+    <ion-card-content>
+      <ion-label color=stable text-wrap>
+        <div text-center style="padding-top: 2%">
+          <p>
+            This app demonstrates how to use the AeroGear DataSync and other Mobile Services
+            tasks such Data Exchange, Identity Management, Device Security, Metrics Capture, Push Notifications and
+            more.
```

Comment:
can we change this to the origin text we had in the cordova showcase app:

"This app demonstrates how to use the AeroGear Mobile services, backed by OpenShift to perform a wide range of common mobile tasks such as Data Synchronisation, Identity Management, Device Security, Metrics Capture, Push Notifications and more."